analyzer/src/main/kotlin/managers/Pub.kt

```diff
@@ -72,6 +72,8 @@ import org.ossreviewtoolkit.utils.ort.showStackTrace
 import org.semver4j.RangesList
 import org.semver4j.RangesListFactory
 
+private const val OPTION_PUB_DEPENDENCIES_ONLY = "pubDependenciesOnly"
```

Comment:
Ah, this reminds my I should have added a constant for `"gradleVersion"`, too. Thanks 😁

Reply:
See https://github.com/oss-review-toolkit/ort/pull/6641.